.github/workflows/release-provider.yml

```
@@ -50,7 +50,7 @@ jobs:
         uses: crazy-max/ghaction-import-gpg@v3
         with:
           gpg-private-key: ${{ secrets.GPG_PRIVATE_KEY }}
-          passphase: ${{ secrets.PASSPHRASE }}
+          passphrase: ${{ secrets.PASSPHRASE }}
       - name: Release provider
         uses: goreleaser/goreleaser-action@v1
```

In the master branch, we use `v2.4.1` 🤔 
```suggestion
        uses: goreleaser/goreleaser-action@v2.4.1
```

---

I think i missed another PR to cherry pick 🍒  🙇  thank you 🙇